shared/desktop/renderer/preload-main.shared.desktop.tsx

```diff
@@ -69,12 +69,38 @@ const darwinCopyToChatTempUploadFile = isDarwin
       throw new Error('unsupported platform')
     }
 
+// Expose native file picker to components.
+// Improved experience over HTML <input type='file' />
+const openFile = async (options: KBElectronOpenProperties) => {
```

Comment:
lets name them so it closer to whats in electron aka
`KB.electron.dialog.showOpenDialog`